### ISSUE TITLE
Bug: When another attendee declines a meeting, it appears as canceled

### ISSUE THREAD

**Issue by joshuarudd** [leits/MeetingBar#224]
**App version**
3.1.1

**Describe the bug**
When another attendee declines a meeting, the meeting appears as canceled in the menubar (strikethrough), and states it is canceled in the dropdown meeting details.

In addition, the details section shows that _I_ am the one that declined, even though the name next to it was a different attendee.

**To Reproduce**
1. Create or accept invite to a Google Calendar meeting
2. Another attendee declines the meeting (in the most recent case, it was the meeting organizer that declined)

**Expected behavior**
When another attendee declines the meeting, the meeting does not appear as canceled for me.

**Screenshots**
![2021-03-24 09 31 04 - CleanShot@2x](https://user-images.githubusercontent.com/23268/112319958-f2158380-8c84-11eb-849d-7c7064710ff6.png)

**Desktop (please complete the following information):**
 - macOS version: 11.2.3

**Additional context**
None.

**Comment by joshuarudd** [comment]
Okay, small update here. After posting this I declined the event, but now it shows that I've accepted it (it thinks I'm the remaining attendee on the meeting).

So it seems that the root of the issue is that MeetingBar doesn't really know _which_ attendee I am on a meeting, and is reflecting the accepted status of other attendees for me.

![2021-03-24 10 41 26 - CleanShot@2x](https://user-images.githubusercontent.com/23268/112329837-eaa6a800-8c8d-11eb-98f8-25e43de9289f.png)
